### ISSUE TITLE
Yarn berry with version 0.20.20 and 0.20.21 fails, works with 0.20.19

### ISSUE THREAD

**Issue by christophehurpeau** [TypeStrong/typedoc#1496]
## Search terms

yarn berry shiki-themes

## Expected Behavior

No error

## Actual Behavior

```
Error: typedoc tried to access shiki-themes, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: shiki-themes (via "shiki-themes")
Required by: typedoc@virtual:d40d89e6c692773fbe2cca237d25d8715b0eb161f2bc2020d9ef4bc61239aa41f0bb4b7e0d0d90e3ca52293e9854a9aded5f139092d4de5ddba5af210acd328e#npm:0.20.21 (via /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/utils/options/sources/)

Require stack:
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/utils/options/sources/typedoc.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/utils/options/sources/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/utils/options/options.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/utils/options/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/utils/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/models/reflections/project.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/models/reflections/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/models/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/converter/context.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/converter/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/lib/application.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/dist/index.js
- /home/chris/libs/nightingale/.yarn/$$virtual/typedoc-virtual-3139956e90/0/cache/typedoc-npm-0.20.21-de3c8089e2-9a04d78708.zip/node_modules/typedoc/bin/typedoc
```

## Environment

-   Typedoc version: 0.20.21
-   TypeScript version: 4.1.3
-   Node.js version: 14.15.1
-   OS: Ubuntu

**Comment by Gerrit0** [comment]
Fixed + released in 0.20.22 :)

**Comment by christophehurpeau** [comment]
Thank you !
